optimum/habana/checkpoint_utils.py

```diff
@@ -94,7 +94,7 @@ def model_on_meta(config):
     """
     Checks if load the model to meta.
     """
-    return config.model_type in ["bloom", "llama", "falcon", "mixtral", "qwen2"]
+    return config.model_type in ["bloom", "llama", "falcon", "qwen2"]
```

Comment:
i guess this removal is due to the w1 and w3 merge when loading the model from ckpt. but that is irrelevant to static moe and static moe can still have benefit from loading to meta. anyway to separate those 2 cases?